cmd/juju/commands/debuglog.go

```diff
@@ -187,9 +195,12 @@ func (c *debugLogCommand) Init(args []string) error {
 		}
 		c.params.Level = level
 	}
-	if c.tail && c.notail {
+	if c.tail && c.noTail {
```

Comment:
Could have a test for this too...

Reply:
We do. https://github.com/juju/juju/blob/develop/cmd/juju/commands/debuglog_test.go#L111-L112